src/webui/CMakeLists.txt

```diff
@@ -1,38 +1,44 @@
 add_library(qbt_webui STATIC
-# headers
-api/apicontroller.h
-api/apierror.h
-api/appcontroller.h
-api/authcontroller.h
-api/freediskspacechecker.h
-api/isessionmanager.h
-api/logcontroller.h
-api/rsscontroller.h
-api/searchcontroller.h
-api/synccontroller.h
-api/torrentscontroller.h
-api/transfercontroller.h
-api/serialize/serialize_torrent.h
-webapplication.h
-webui.h
+    # headers
+    api/apicontroller.h
+    api/apierror.h
+    api/appcontroller.h
+    api/authcontroller.h
+    api/freediskspacechecker.h
+    api/isessionmanager.h
+    api/logcontroller.h
+    api/rsscontroller.h
+    api/searchcontroller.h
+    api/synccontroller.h
+    api/torrentscontroller.h
+    api/transfercontroller.h
+    api/serialize/serialize_torrent.h
+    webapplication.h
+    webui.h
 
-# sources
-api/apicontroller.cpp
-api/apierror.cpp
-api/appcontroller.cpp
-api/authcontroller.cpp
-api/freediskspacechecker.cpp
-api/logcontroller.cpp
-api/rsscontroller.cpp
-api/searchcontroller.cpp
-api/synccontroller.cpp
-api/torrentscontroller.cpp
-api/transfercontroller.cpp
-api/serialize/serialize_torrent.cpp
-webapplication.cpp
-webui.cpp
+    # sources
+    api/apicontroller.cpp
+    api/apierror.cpp
+    api/appcontroller.cpp
+    api/authcontroller.cpp
+    api/freediskspacechecker.cpp
+    api/logcontroller.cpp
+    api/rsscontroller.cpp
+    api/searchcontroller.cpp
+    api/synccontroller.cpp
+    api/torrentscontroller.cpp
+    api/transfercontroller.cpp
+    api/serialize/serialize_torrent.cpp
+    webapplication.cpp
+    webui.cpp
 )
 
-qbt_target_sources(qBittorrent PRIVATE www/webui.qrc)
+target_sources(qbt_webui
+    INTERFACE
+        www/webui.qrc
+)
 
-target_link_libraries(qbt_webui PUBLIC qbt_base)
+target_link_libraries(qbt_webui
+    PRIVATE
+        qbt_base
```

Comment:
:-1: to split it (and all other short fragments).

Reply:
All such instances fixed.